index.html

```diff
@@ -17,6 +17,7 @@ <h1>0data Hello World</h1>
             <li><a href="./fission/">Fission</a></li>
             <li><a href="./remotestorage/">remoteStorage</a></li>
             <li><a href="./solid/">Solid</a></li>
+            <li><a href="./solid2/">Solid2</a></li>
```

Comment:
Since we're going to have more hello worlds, maybe we should describe them briefly in the index? Maybe something like:

- Solid (Vanilla JS)
- Solid (solid-file-client)
- Solid (solid-client)
- Solid (soukai)
- etc.

If we do that, the one with Vanilla JS can be called `solid`, and the other ones using the name of the library. e.g. `solid-solid-file-client`, `solid-solid-client`, etc.